src/DETHRACE/common/loading.c

```diff
@@ -842,7 +842,7 @@ tS8* ConvertPixToStripMap(br_pixelmap* pThe_br_map) {
     temp_strip_image = BrMemAllocate(pThe_br_map->row_bytes * pThe_br_map->height, kMem_strip_image);
     current_size = 2;
 
-    *temp_strip_image = pThe_br_map->height;
+    *(uint16_t *)temp_strip_image = pThe_br_map->height;
```

Comment:
We're trying to use BRender data types in the dethrace code base.
```suggestion
    *(br_uint16_t*)temp_strip_image = pThe_br_map->height;
```